### PR TITLE
add parameter create_short_label

### PR DESCRIPTION
**Pull request by WillMcDGSK**
@paltusplintus or @JonathanDeacon Please review!
@PranjaliParnerkar FYI
Small PR for [this PBI](https://dev.azure.com/DevOps-RD/Clinical%20Linked%20Data%20Experiments/_sprints/taskboard/Clinical%20Linked%20Data%20Experiments%20Team/Clinical%20Linked%20Data%20Experiments/Sprint%209.6?workitem=991143). It adds a parameter to generate_excel_based_model() that will control the creation of short_label property on relationship nodes.

### CODE REVIEW

File: cdisc_model_managers/cdisc_model_manager.py [hunk 1]
```diff
@@ -98,13 +99,17 @@ def generate_excel_based_model(self, label_terms: bool = False, create_term_inde
                    d.Description + ' ' + v.Label
                ELSE
                    v.Label
-               END                
-           SET v.short_label =             
+               END
+           SET v.shot_label = CASE WHEN $short_label_bool
```
Comment (JonathanDeacon) [inline cdisc_model_managers/cdisc_model_manager.py:103]: Typo: `v.shot_label`, should be `v.short_label`

File: cdisc_model_managers/cdisc_model_manager.py [hunk 2]
```diff
@@ -9,12 +9,13 @@ def __init__(self, *args, **kwargs):
...
     ## ---------------------------- Generate model from excel SDTM spec ----------------------------- ##
-    def generate_excel_based_model(self, label_terms: bool = False, create_term_indexes: bool = False):
+    def generate_excel_based_model(self, label_terms: bool = False, create_term_indexes: bool = False, create_short_label: bool = False):
         """
         Run ExcelStandardLoader.load_standard() to prepare metadata from excel (and SDTM ontology form GitHub)
         in Neo4j
-        :param create_term_indexes: weather to create indexes for each Class that HAS_CONTROLLED_TERM (can be done later
+        :param create_term_indexes: Whether to create indexes for each Class that HAS_CONTROLLED_TERM (can be done later
         during reshaping)
+        :param create_term_indexes: Whether to create the short_label property on Relationship (Variable) nodes
```
Comment (JonathanDeacon) [inline cdisc_model_managers/cdisc_model_manager.py:18]: Typo: Should be `:param create_short_label: `

File: cdisc_model_managers/cdisc_model_manager.py (hunk 1 above)
Comment (JonathanDeacon) [inline cdisc_model_managers/cdisc_model_manager.py]: Typo: `v.shot_label`, should be `v.short_label`

**Review by JonathanDeacon — CHANGES_REQUESTED** [review]
Spotted a few minor typos. I'm also a bit confused by the PBI which mentioned "disable creation of short_label on Relationship nodes" specifically? Probably best to double check with  @paltusplintus

**Comment by WillMcDGSK** [comment]
> Spotted a few minor typos. I'm also a bit confused by the PBI which mentioned "disable creation of short_label on Relationship nodes" specifically? Probably best to double check with @paltusplintus

When we load data in the via the model manager's `generate_excel_based_model()` sometime the variables become relationships instead of classes. In this case they would also have the same properties as classes including short_label, which causes other issues down the line. We don't need the short_label prop on the relationships nodes, so we can just not add them here. 
I have realized that I also included this change to the 'Create Class from Variable' section as well, which was a mistake! I will remove that now.

**Comment by JonathanDeacon** [comment]
> @JonathanDeacon When we load data in the via the model manager's `generate_excel_based_model()` sometime the variables become relationships instead of classes. In this case they would also have the same properties as on classes including short_label, which causes other issues down the line. We don't need the short_label prop on the relationships nodes, so we can just not add them here. I have realized that I also included this change to the 'Create Class from Variable' section as well, which was a mistake! I will remove that now.

@WillMcDGSK Oh that makes sense! I was confused at first why the changes were in create class from variable and create relationship from variable so I assumed the PBI hadn't been updated.

**Review by JonathanDeacon — APPROVED** [review]
Looks good!

**Pull request merged**